### ISSUE TITLE
Hlormir Wine-sot's unrelenting pestering about his axe [BUG]

### ISSUE THREAD

**Issue by nalal** [TES3MP/TES3MP#240]
When attempting to complete the quest for Hlormir Wine-sot to recover his axe from a witch he will force a dialogue in which he asks for his axe back however he doesn't stop asking and instead will continue to ask you for the axe constantly and essentially loop his request permanently until either the axe is removed from the player's inventory or the NPC dies

**Comment by ghost** [comment]
Adding some more info here...

The quest works in OpenMW.  The relevant ingame script is the local script "hlormarScript" on Hlormar Wine-sot.  It continuously checks whether the player has "cloudcleaver_unique" in their inventory.  If so, it ForceGreeting's the player:
```...
if ( greetonce == 0 )
    if ( Player->GetItemCount "cloudcleaver_unique" > 0 )
        if ( GetDistance Player <= 512 )
            ForceGreeting
            ;set greetonce to -1
        endif
    endif
endif
```
GIF of working behavior in OpenMW: https://gfycat.com/ScalyPleasantCoot
GIF of currently broken behavior (on build 0714): https://gfycat.com/ShamelessResponsibleLabradorretriever

**Comment by davidcernat** [comment]
Fixed by 2f551ab at the behest of #249.
